src/components/Card/CardImage/CardImage.js

```diff
@@ -0,0 +1,22 @@
+import "./CardImage.css";
+import defaultImage from "../../../assets/images/card-default.png";
+import star from "../../../assets/images/folder/star.svg";
+
+const CardImage = ({ items, isZoomedIn }) => {
+  const { imageSource, image_source: image_sourse } = items;
```

Comment:
P2:
왜 이렇게 작업하셨는지 모르겠어요.

```suggestion
  const { imageSource, image_source } = items;
```